sa/storage-authority.go

```diff
@@ -232,6 +232,50 @@ func (ssa *SQLStorageAuthority) GetLatestValidAuthorization(registrationID int64
 	return ssa.GetAuthorization(auth.ID)
 }
 
+// GetValidAuthorizations returns the latest authorization object for all
```

Comment:
This comment is not accurate, I think. This function only returns some authorization object that is both valid and unexpired for each name, if such an object exists.